views/content/edit.php

```diff
@@ -107,6 +107,93 @@ class="input-mini timeentry editable" type="text"
         </select>
       </div>
     </div>
+      <?php
+      $problemTypeConfig = $config['problem_type'];
```

Comment:
To avoid a notice for array key does not exist, I'd do this as:

```php
if (array_key_exists('problem_type', $config)) {
    $problemTypeConfig = $config['problem_type'];
    // ... the rest of the if body
```

Reply:
Thanks I will do this change